applications/nrf_desktop/configuration/nrf52810_pca20045/app_ZDebug.conf

```diff
@@ -36,6 +36,7 @@ CONFIG_DESKTOP_BLE_DIRECT_ADV=n
 CONFIG_DESKTOP_BLE_FAST_ADV=y
 CONFIG_DESKTOP_BLE_SWIFT_PAIR=y
 CONFIG_DESKTOP_BLE_SHORT_NAME="Mouse nRF52"
+CONFIG_DESKTOP_BLE_SECURITY_FAIL_TIMEOUT_S=10
```

Comment:
I think it would be better to remove `_S` postfix. Don't recall seeing it so far in Kconfig...

Reply:
Sebastian Boe asked me about adding this one, I think we should leave it.